components/ExercisePreview.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styles from '../scss/exercisePreview.module.scss'
+
+type exerciseProps = {
```

Comment:
NIT: What about naming it `Exercise` since it's not a prop for a component but a type of prop.